.travis.yml

```diff
@@ -5,5 +5,5 @@ env:
     - DOCKER_IMAGE=fedora:24
     - DOCKER_IMAGE=fedora:25
     - DOCKER_IMAGE=fedora:rawhide
```

Comment:
I don't have preference, but the the situation with `travis.yml` is the same as with `Makefile.docker` - it will have to be updated with every new Fedora. If it is just added as additional image, we will be most probably testing one Fedora version twice, but on the other hand we won't miss new Fedora if we don't update the file on time. If we replace Fedora versions with just `fedora:latest`, we won't test rebase-helper on "latest stable version" - 1. I'm OK also if no change will be done here, but it seemed right to consolidate Makefile and config file for Travis.